megamek/i18n/megamek/client/messages_de.properties

```diff
@@ -273,6 +273,7 @@ CommonMenuBar.FileMenu=Datei
 CommonMenuBar.fileUnitsClear=Löschen
 CommonMenuBar.fileUnitsOpen=Öffnen...
 CommonMenuBar.fileUnitsSave=Speichern...
+CommonMenuBar.fileUnitsBrowse=Verfügbare Einheiten...
```

Comment:
This may cause I/O issues in future (and makes it impossible to review), and as such you'll want to use the unicode instead of any special characters.

`\u01EE` over `Ǯ`, for example.

This is one of the places MM's resources has issues, and its broken hard in MHQ before.